module/Company/view/company/admin/index.phtml

```diff
@@ -24,110 +24,141 @@
                         <tr>
-                            <th>
-                                <?= $this->translate('Name') ?>
-                                <a onclick="Company.Admin.sortCompaniesByName(1)"><span class="fas fa-arrow-up"></span></a>
-                                <a onclick="Company.Admin.sortCompaniesByName(-1)"><span class="fas fa-arrow-down"></span></a>
-                            </th>
-                            <th>
-                                <?= $this->translate('Jobs') ?>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(1, 1)"><span class="fas fa-arrow-up"></span></a>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(-1, 1)"><span class="fas fa-arrow-down"></span></a>
-                            </th>
-                            <th>
-                                <?= $this->translate('Active jobs') ?>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(1, 2)"><span class="fas fa-arrow-up"></span></a>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(-1, 2)"><span class="fas fa-arrow-down"></span></a>
-                            </th>
-                            <th>
-                                <?= $this->translate('Banner active') ?>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(1, 3)"><span class="fas fa-arrow-up"></span></a>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(-1, 3)"><span class="fas fa-arrow-down"></span></a>
-                            </th>
-                            <th>
-                                <?= $this->translate('Featured in language') ?>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(1, 4)"><span class="fas fa-arrow-up"></span></a>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(-1, 4)"><span class="fas fa-arrow-down"></span></a>
-                            </th>
-                            <th>
-                                <?= $this->translate('Expired packages') ?>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(1, 5)"><span class="fas fa-arrow-up"></span></a>
-                                <a onclick="Company.Admin.sortCompaniesByColumn(-1, 5)"><span class="fas fa-arrow-down"></span></a>
-                            </th>
-                            <th>
-                                <?= $this->translate('Actions') ?>
-                            </th>
+                            <td><?= $label->getName(); ?> (<a href=<?= $editUrl; ?>><?= $this->translate('edit')?></a>)</td>
                         </tr>
+                    <?php endforeach; ?>
+                    </tbody>
+                    <tfoot>
+                    <tr>
+                        <th><?= $this->translate('Categories') ?></th>
+                    </tr>
+                    </tfoot>
```

Comment:
This should be `Labels` and not categories. However, is there what is the reason we should have an additional table footer when the name is already in the table header?

The same applies to the table footer for the `Categories` section.

Reply:
No, there is no reason. Just being consistent with previous designs